packages/astro/src/core/cookies/cookies.ts

```diff
@@ -117,11 +119,13 @@ class AstroCookies implements AstroCookiesInterface {
 			}
 		}
 
-		const values = this.#ensureParsed(options);
+		const decode = options?.decode ?? decodeURIComponent
```

Comment:
I don't understand this part. Why do we use `decodeURIComponent`? The previous code didn't use it, and the PR doesn't mention it. What am I missing?

Reply:
It's the default in the `cookie` library if you don't pass in anything. That's why we pass an identity function to `parse`

Reply:
I've added a comment